packages/vuetify/src/components/VDataTable/VDataTableVirtual.tsx

```diff
@@ -243,6 +247,8 @@ export const VDataTableVirtual = genericComponent<new <T extends readonly any[],
                       <td colspan={ columns.value.length } style={{ height: 0, border: 0 }}></td>
                     </tr>
                   </tbody>
+                  { slots.tbody?.(slotProps.value) }
```

Comment:
I would expect this to be something like:
```tsx
{ slots.tbody?.(slotProps.value) ?? (
  <tbody>...</tbody>
)}
```

Reply:
@johnleider I copeid this verbatum from https://github.com/vuetifyjs/vuetify/blob/2b3ccee8cf538f95278797a1b47a85898f35197d/packages/vuetify/src/components/VDataTable/VDataTable.tsx#L258